test/server/data/regression/gh-599/testfile.js

```diff
@@ -0,0 +1,23 @@
+import createCallsiteRecord from '../../../../../lib/errors/get-callsite.js';
+
+fixture `Test`
+    .page `http://example.com`;
+
+var obj = {
+    method1: () => {
+        throw new Error('Error on line - ' + (createCallsiteRecord('method1').lineNum + 1));
```

Comment:
Oh, just found out that you use `lineNum`, it's not a part of public API, it's better to render callsite and check that rendered string contains `> 19 |        .method1()`